src/initialization/MOM_state_initialization.F90

```diff
@@ -404,6 +408,23 @@ subroutine MOM_initialize_state(u, v, h, tv, Time, G, GV, US, PF, dirs, &
   if (use_temperature .and. use_OBC) &
     call fill_temp_salt_segments(G, GV, OBC, tv)
 
+  ! Calculate the initial surface displacement under ice shelf
+
+  if (new_sim) then
+    call get_param(PF, mdl, "DEPRESS_INITIAL_SURFACE", depress_sfc, &
```

Comment:
Please read these two parameters (but do not necessarily log them), even if this is not a new run. Otherwise we will be unable to require that all parameters in input files be read to test for misspelled or inappropriate parameters in the the input files, or else we would have to change the input files between new runs and runs from restarts, neither of which is desirable.